plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/HeteroDescribableConfigurator.java

```diff
@@ -19,7 +20,10 @@
 import io.vavr.collection.HashMap;
 import io.vavr.collection.Stream;
 import io.vavr.control.Option;
+
```

Comment:
Remove this blank line to make findbugs happy

Reply:
Imports settings are A-Z, no star imports, static last with a separation between imports and static imports